user-guide/Standard_Apps/Monitoring_Solutions/Ping_Monitoring/Tutorial_Apps_Ping_Monitoring_App.md

```diff
@@ -0,0 +1,148 @@
+
+1. Open the Ping Monitoring tool.
+
+   This is what the application will look like:
+
+   ![Ping Monitoring UI](~/user-guide/images/brand_new_destinations-PingMonitoring.png)
+
+> [!TIP]
+> See also: [Ping Monitoring UI overview](xref:Ping_Monitoring_UI)
+
+## Step 2: Create a new group
+
+Groups are used to organize the destinations configured in the application. Destinations that belong to the same group share the same properties that differentiate them from other destinations.
+
+1. In the Ping Monitoring tool, go to the *Groups* page.
+
+1. In the header bar, click *Create Group*.
+
+   This will open a pop-up window.
+
+1. Enter group name *Demo*, then click *Next*.
+
+1. Populate the three property fields with the values `Americas`, `Gold`, and `CustomerXYZ`, respectively, and then click *Next*.
+
+1. Set the interval to `60`.
+
+1. Click *Save*.
+
+![new group](~/user-guide/images/Create_New_Group-PingMonitoring.gif)
+
+## Step 3: Create a new destination
+
+A destination is a point in the network you want to test against. The Ping Monitor application allows you to easily manage hundreds and even thousands of destinations to monitor the availability of different devices and services in your network.
+
+In this step, you will create three destinations using the following addresses accessible on the public internet. The first value corresponds with the address, and the second is the description:
+
+- 8.8.8.8 (Google DNS)
+- skyline.be (Skyline website)
+- dataminer.services (DataMiner Services)
+
+Carry out the following steps for each one of the destinations above:
+
+1. In the Ping Monitoring tool, go to the *Destinations* page.
+
+1. In the header bar, click *Add Destination*.
+
+1. Select the *Demo* group.
+
+1. Specify the address and description in the corresponding boxes.
+
+1. Click *Add Destination*.
+
+![new destination](~/user-guide/images/Create_New_Destination-PingMonitoring.gif)
+
+> [!TIP]
+> See also: [Adding a new destination](xref:Ping_Monitoring_managing_groups_destinations#adding-a-new-destination).
+
+## Step 4: Analyze the results
+
+When you have configured the destinations, the application will update to display the results of the latest test cycles.
+
+1. In the *Destinations* table, take a look at the values in the following columns:
+
+   - *Ping Result*: The amount of time passed from the moment DataMiner sent the command to the moment it received the reply from each destination, e.g. *4 ms*.
+   - *Cycle packet loss*: The percentage of lost packets during the last ping cycle, e.g. *0.00 %*. By default, the application sends 4 packets in every cycle, configurable for each destination.
+
+   These values can help you monitor network congestion and detect outages.
+
+1. If you are a member of the DevOps Program, take a screenshot of the *Destinations* table (similar to the one below), ensuring all columns are clearly visible, and send it to TBD to get DevOps Points.
```

Comment:
Previously, this mentioned to upload the screenshot, but it didn't say where. For most tutorials, we ask to send an email, so I think we'd best do that in this case too, but where should it be sent to? (See for example [Anomaly Detection tutorial](https://docs.dataminer.services/user-guide/Basic_Functionality/Alarms/Working_with_alarms/Advanced_analytics_features/Tutorials/Anomaly_Detection_Tutorial.html#step-7-final-exercise).)